server/journeys/screen/check-answers/controller.ts

```diff
@@ -10,8 +11,29 @@ export class ScreenCheckAnswersController extends BaseJourneyController {
     })
   }
 
-  checkSubmitToAPI: RequestHandler = async (_req, _res, next) => {
-    // Not implemented yet
+  checkSubmitToAPI: RequestHandler = async (req, res, next) => {
+    const screening = req.journeyData.saferCustodyScreening!
+    try {
+      await this.createScreeningOutcome(req, {
+        outcomeTypeCode: screening.outcomeType!.code,
+        date: new Date().toISOString().substring(0, 10),
```

Comment:
i expect this to come up often so i have a `todayString()` util function in datetimeUtils for this

Reply:
glorious